README.adoc

```diff
@@ -23,52 +23,54 @@ Refer to link:https://docs.cloudbees.com/docs/cloudbees-platform/latest/github-a
 | `name`
 | String
 | Yes
-| The name of the artifact to send to the CloudBees platform for artifact traceability purposes.
+| The name of the artifact reported to the CloudBees platform.
 
-| `version`
+| `url`
 | String
 | Yes
-| The version of the artifact to send to the CloudBees platform for artifact traceability purposes.
+| The URL where the artifact version is located.
+For example, `docker.io/myapp/myimg:1.0.0`.
 
-| `url`
+| `version`
 | String
 | Yes
-| The URL where the artifact version can be pulled for deployment.
+| The version of the artifact to send to the CloudBees platform for artifact traceability purposes.
 
 | `cloudbees-url`
 | String
 | No
 | The CloudBees platform URL. The default value is `"https://api.cloudbees.io"`.
 
-| `digest`
+| `commit`
```

Comment:
can we move `commit` down to be right before `ref` so that the code commit fields are listed together `commit`, `ref` and `repository-url`?

Reply:
we have been doing everything alphabetically, but if you want we can.

Reply:
oh I didn't realize there was a particular order being followed as a convention. Then for now, lets stay with alphabetical.